rocAL/include/meta_data/meta_data.h

```diff
@@ -135,18 +142,50 @@ class MetaData {
     MetaDataInfo _info;
 };
 
+class AsciiValue : public MetaData {
+   public:
+    AsciiValue() = default;
+    AsciiValue(AsciiValues ascii_values) {
+        _ascii_values = std::move(ascii_values);
+    }
+    void set_ascii_values(AsciiValues ascii_values) override { _ascii_values = std::move(ascii_values); }
+    AsciiValues& get_ascii_values() override { return _ascii_values; }
+    // Not Implemnented
+    void set_labels(Labels label_ids) override { THROW("Not Implemented") }
```

Comment:
I don't recomment this way of using MetaData class with most function not implemented. We need to discuss